src/resources/nfts/simplehash/utils.ts

```diff
@@ -136,16 +134,28 @@ export function simpleHashNFTToUniqueAsset(nft: SimpleHashNFT): UniqueAsset {
     external_link: nft.external_url,
     familyImage: collection.image_url,
     familyName: collection.name,
+    floorPriceEth: floorPrice?.value
```

Comment:
added floor price attribute bc simplehash provides it and it would otherwise have to be fetched separately in the nft expanded state